Makefile

```diff
@@ -4,21 +4,18 @@ SHELL := bash
 BUILD_DIR := build
 SRC_DIR := src
 
-YOSYS_CONFIG := yosys-config
-CFLAGS ?= -O0 -Wall -lstdc++
-LDFLAGS :=  
+YOSYS_PATH := $(realpath $(HOME)/data/yosys)
```

Comment:
Some Day it may be nice to automatically find the path, but I'm not hugely worried about needing to specify an environment variable right now.